src/contexts/auth-context/auth-context.js

```diff
@@ -0,0 +1,34 @@
+import { createContext, useContext, useReducer, useEffect } from "react";
+import { authReducer } from "../../reducers";
+
+const initialAuthValue = {
+  isAuth: false,
+  user: "",
+  encodedToken: "",
+};
+
+const AuthContext = createContext(initialAuthValue);
```

Comment:
dispatch function should also be included in this, since you are passing that in the value {}